python/perf-kernels/layernorm.py

```diff
@@ -227,13 +243,12 @@ def parse_args():
     )
 
     parser.add_argument('-M', "--M_start", default="1", type=int)
-    parser.add_argument('-Ms', "--M_step", default="2", type=int)
-    parser.add_argument('-Me', "--M_end", default="512", type=int)
-    parser.add_argument('-Mb', "--M_benchmark", default=False, type=bool)
+    parser.add_argument('-Ms', "--M_step", default="0", type=int)
+    parser.add_argument('-Me', "--M_end", default="0", type=int)
 
-    parser.add_argument('-N', "--N_start", default="1024", type=int)
-    parser.add_argument('-Ns', "--N_step", default="2048", type=int)
-    parser.add_argument('-Ne', "--N_end", default="65536", type=int)
+    parser.add_argument('-N', "--N_start", default="65536", type=int)
```

Comment:
[Non-blocking] I think default `argparse` values do not necessarily need to be strings, you can use `65536` instead of `"65536"`, `0` instead of `"0"` and so on...